src/commands/publish/upload_form/project_assets.rs

```diff
@@ -0,0 +1,37 @@
+use super::binding::Binding;
+use super::file::File;
+use super::wasm_module::WasmModule;
+
+#[derive(Debug)]
+pub struct ProjectAssets {
+    pub script_path: String,
+    pub wasm_modules: Vec<WasmModule>,
```

Comment:
deafult to none?

Reply:
idk now, i think this was a thing i said while we were CRing together and it doesn't buy much.